services/src/handlers/wms.rs

```diff
@@ -169,6 +170,19 @@ async fn get_map<C: Context>(
         .initialize(&execution_context)
         .context(error::Operator)?;
 
+    // handle request and workflow crs matching
+    let spatial_ref = initialized.result_descriptor().spatial_reference();
+    // TODO: use a default spatial reference if it is not set?
```

Comment:
also here, default to 4326

Reply:
The GeoServer docs state crs is a required parameter. So it makes no sense to allow it to be empty. 
https://docs.geoserver.org/stable/en/user/services/wms/reference.html